components/Zakat/ZakatCalculator.vue

```diff
@@ -0,0 +1,50 @@
+<template>
+  <div>
+    <h3>{{ $t('ZakatPage.Calculator.title') }}</h3>
+    <h4>{{ $t('ZakatPage.Calculator.title2') }}</h4>
+    <v-text-field
+      v-model="inputSum"
+      placeholder="Сумма ваших денег ₽"
+      filled
+      rounded
+      type="Number"
+      dense
+    />
+    <p class="font-weight-regular">
+      {{ $t('ZakatPage.Calculator.nisabtoday') }}  {{ nisabSum }} ₽
+    </p>
+    <p class="font-weight-regular">
+      {{ $t('ZakatPage.Calculator.bank') }} {{ goldPrice }} ₽/г золото
+    </p>
+    <br>
+    <div v-if="nisabSum<=inputSum">
+      <h4>{{ $t('ZakatPage.Calculator.yourzakat') }} {{ 0.025*inputSum }}</h4>
+      <v-btn rounded color="primary" dark>
+        {{ $t('ZakatPage.Calculator.pay') }}
+      </v-btn>
+    </div>
+    <div v-else>
+      <p class="font-weight-regular">
+        {{ $t('ZakatPage.Calculator.nopay') }}
+      </p>
+    </div>
+  </div>
+</template>
+
+<script>
+import Vue from 'vue';
+
+export default Vue.extend({
+  data () {
+    return { goldPrice: 0, nisabSum: 0, inputSum: null };
+  },
+  async mounted () {
+    const date = new Date(Date.now()).toLocaleString().split(',')[0];
```

Comment:
The `new Date()` constructor already returns the current date, no need to use `Date.now()`.

Using [`toLocaleString()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleString) without specifying a locale makes it use the current locale, which has a different format for each person. It is better to use [`.getDate()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getDate) and [`.getMonth()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getMonth) (note that `date` ranges from 1 to 31, while `month` ranges from 0 to 11)

```suggestion
    const now = new Date();
    const day = now.getDate();
    const month = now.getMonth() + 1;
    const year = now.getFullYear();
    const today = `${day.toString().padStart(2, '0')}/${month.toString().padStart(2, '0')}/${year}`;
```